### DISCUSSION TITLE
Не качает с https://2.mintmanga.one/

### DISCUSSION THREAD

**Discussion by dzero6** [lirix360/ReadmangaGrabber#107] [Q&A]
Авторизация проходит успешно, список глав выводится и начинается закачка, но качает пустые файлы. Пишет "Ошибка при скачивании страницы" :(
![Снимок](https://github.com/user-attachments/assets/eef08479-a6a7-48da-9d28-35ea7f36d7ac)

**Comment by lirix360** [comment]
> Авторизация проходит успешно, список глав выводится и начинается закачка, но качает пустые файлы. Пишет "Ошибка при скачивании страницы" :( 

Исправлено в 2.2.5
